src/battle/anim/ghost.c

```diff
@@ -189,3 +202,402 @@ const struct SpriteTemplate gSpriteTemplate_83DAF98 =
     .affineAnims = gDummySpriteAffineAnimTable,
     .callback = sub_80DF49C,
 };
+
+void sub_80DDB6C(struct Sprite *sprite) {
```

Comment:
Opening curly braces should be on their own line for both functions and `if`, `else`, `switch`, etc. blocks.